toolchain/check/testdata/function/definition/extern.carbon

```diff
@@ -19,7 +19,7 @@ extern fn F() {}
 library "def_for_extern_decl" api;
 
 extern fn F();
-// CHECK:STDERR: fail_def_for_extern_decl.carbon:[[@LINE+7]]:1: ERROR: Redeclaring `extern` function `F` as non-`extern`.
+// CHECK:STDERR: fail_def_for_extern_decl.carbon:[[@LINE+7]]:1: ERROR: Defining `extern fn F` is disallowed.
```

Comment:
Is this change in the phrasing intentional? I think I'd have a hard time understanding the new version if I encountered it in ordinary usage.

Reply:
The previous phrasing was bothering me because it didn't read as well for me when working with `class`. Maybe collapsing this diagnostic into the other `extern` check will work better?